inference-engine/src/gna_plugin/layers/gna_permute.hpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+#pragma once
+#include <vector>
+#include <list>
+#include <utility>
+#include "gna_plugin_log.hpp"
+
+namespace GNAPluginNS {
+template <class T>
+class PermuteSequence {
```

Comment:
please restore unit tests for permute sequence as i remember there lots of cases tested